### DISCUSSION TITLE
[ Error ] Test 절대 경로 이슈

### DISCUSSION THREAD

**Discussion by tkyoun0421** [Upjuyanolja/Upjuyanolja_FE#59] [Q&A]
현재 테스트를 위해 파일을 만들던 중 저희 프로젝트에서 alias를 사용하여 절대경로로 이용하고 있는데
테스트 파일에서는 절대 경로를 인식하지 못하는 이슈가 있습니다.

![image](https://github.com/Upjuyanolja/Upjuyanolja_FE/assets/98436988/b361817c-d67b-4be4-853a-4ae2b32707a4)

**[ 절대 경로를 읽어오지 못하는 모습 ]**

---

![image](https://github.com/Upjuyanolja/Upjuyanolja_FE/assets/98436988/cbf8c1f3-b328-4fa8-90c8-1ed93b227302)

**[ 상대 경로는 에러가 없는 모습 ]**

이에 대해 고민하고 있는데 혹시 해결 방법을 아시는 분은 말씀해주시면 감사하겠습니다 :)

p.s) 고치려다가 너무 모르겠어서 이러다 다른 것도 못할까봐 일단 재끼고 퍼블리싱 먼저 하고 있는 상태입니다.

**Comment by YongYong21** [comment]
저도 저 부분이 왜 저렇게 되는지 몰라서 상대경로로 하거나 해당 부분을 가지고 와서 테스트 코드를 작성했었습니다.

해당 부분에 대한 에러 해결방안이 나오지 않는다면 멘토님께 물어보는 게 좋을 것 같습니다 ㅠㅠ

**Comment by gahyuun** [comment]
alias 및 jest 오류 해결 완료
https://github.com/Upjuyanolja/Upjuyanolja_FE/pull/66
